Package.swift

```diff
@@ -36,7 +36,7 @@ let package = Package(
         .package(
             name: "SnapshotTesting",
             url: "https://github.com/pointfreeco/swift-snapshot-testing.git",
-            .upToNextMajor(from: "1.8.0")
+            .upToNextMajor(from: "1.11.0")
```

Comment:
This works for SPM, but unfortunately we can't update the dependency past 1.9 for CocoaPods since SnapshotTesting dropped CP support in 1.10. Unfortunately this means our test suite won't compile right now (not sure why the CI job didn't run for this PR), since it uses the demo app as a host app, which uses CP to pull in its dependencies.